### ISSUE TITLE
ML: VGG Model #1 (24 Layers): Configuration#6: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#270]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs 


Epoch 00030: val_accuracy did not improve from 0.94308

840000/840000 [==============================] - 314s 373us/sample - loss: 0.6529 - accuracy: 0.9380 - val_loss: 0.7048 - val_accuracy: 0.8986

**Comment by hmehryar** [comment]
50 epochs

Epoch 00050: val_accuracy did not improve from 0.94302

840000/840000 [==============================] - 334s 398us/sample - loss: 0.3772 - accuracy: 0.9385 - val_loss: 0.3832 - val_accuracy: 0.9330

**Comment by hmehryar** [comment]
port to excel for overleaf
